packages/app/src/navigation/SidebarNavigationHeader.vue

```diff
@@ -46,6 +46,17 @@
       </div>
     </template>
   </Tooltip>
+  <div
+    v-else
+    class="border-b flex shrink-0 border-gray-900 h-[64px] pl-[20px] items-center"
```

Comment:
Some of these classes could have been pulled out to a `div` surrounding the whole component so they did not have to be duplicated with what is on line 11.  Not going to hold up the PR for that